source/docs/software/advanced-control/state-space/state-space-drivetrain-controllers.rst

```diff
@@ -0,0 +1,48 @@
+State-space Differential Drive Controllers
+==========================================
+
+WPILib includes two new path following controls for differential drivetrains: the ``LTVUnicycleController`` and ``LTVDiffDriveController``. These controllers can be used to accurately track trajectories and account for external disturbances.
+
+Linear Time-Varying Unicycle Controller
+---------------------------------------
+
+Similar to Ramsete, the LTV (Linear Time-Varying) Unicycle Controller generates ``ChassisSpeed`` (C++/Java) references to steer the robot along a trajectory. The system used by this controller has states :math:`\begin{bmatrix}x & y & \theta \end{bmatrix}^T` in the global frame, and linear and angular velocity inputs :math:`\begin{bmatrix}v & \omega \end{bmatrix}^T`. However, this controller utilizes tolerances grounded in reality to pick gains rather than the magical Beta and Zeta gains used by Ramsete.
```

Comment:
You should mention that the linear and angular velocity inputs are in the robot frame.